# HG changeset patch
# User Rik <rik@octave.org>
# Date 1466342634 25200
#      Sun Jun 19 06:23:54 2016 -0700
# Node ID a5924b0cbf4283b007412093b3760fe2029363c8
# Parent  f0303b64e59aabd84f0de08bda0d8d6120ac6857
Return "Screensize" for root graphics property "MonitorPositions" (partial fix bug #48239).

* graphics.in.h: Return "screensize" for root graphics property
"monitorpositions".

diff --git a/libinterp/corefcn/graphics.in.h b/libinterp/corefcn/graphics.in.h
--- a/libinterp/corefcn/graphics.in.h
+++ b/libinterp/corefcn/graphics.in.h
@@ -3155,17 +3155,17 @@ public:
 
     // Programming note: Keep property list sorted if new ones are added.
 
     BEGIN_PROPERTIES (root_figure, root)
       handle_property callbackobject Sr , graphics_handle ()
       array_property commandwindowsize r , Matrix (1, 2, 0)
       handle_property currentfigure S , graphics_handle ()
       string_property fixedwidthfontname , "Courier"
-      array_property monitorpositions r , Matrix (1, 4, 0)
+      array_property monitorpositions r , default_screensize ()
       array_property pointerlocation , Matrix (1, 2, 0)
       double_property pointerwindow r , 0.0
       double_property screendepth r , default_screendepth ()
       double_property screenpixelsperinch r , default_screenpixelsperinch ()
       array_property screensize r , default_screensize ()
       bool_property showhiddenhandles , "off"
       radio_property units U , "inches|centimeters|normalized|points|{pixels}"
     END_PROPERTIES
